debian/postinst

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+mkdir -p /var/lib/ondemand-nginx
+mkdir -p /var/lob/ondemand-nginx
+mkdir -p /var/tmp/ondemand-nginx
+mkdir -p /var/run/ondemand-nginx
+
+mkidir -p /var/lib/ondemand-nginx/config/{puns,apps}
+mkidir -p /var/lib/ondemand-nginx/config/apps/{sys,dev,usr}
+touch /var/lib/ondemand-nginx/config/apps/sys/dashboard.conf
+touch /var/lib/ondemand-nginx/config/apps/sys/shell.conf
+touch /var/lib/ondemand-nginx/config/apps/sys/myjobs.conf
+
+
+# shellcheck disable=SC1091
+[ -e /etc/ood/profile ] && . /etc/ood/profile
+
+export APACHE="/etc/apache2/conf-available/ood-portal.conf"
+/opt/ood/ood-portal-generator/sbin/update_ood_portal --rpm 
+
+cd /etc/apache2/conf-enabled || exit 1
+ln -s ../conf-available/ood-portal.conf . || true
+
+cd /etc/apache2/mods-enabled || exit 1
+ln -s ../mods-available/rewrite.load . || true
+ln -s ../mods-available/lua.load . || true
```

Comment:
Will this lua.load be present by just installing `apache2` or do additional packages need to be added to dependency list?  Some sort of question will exist for some of these other loads. In RPM systems many of these are separate packages.

Reply:
It must becuase it worked, though I'm building the package now to confirm.

My guess is is that it's got every standard package like lua in 'available' and you have to make the symlinks to enable them.

Reply:
Likely, RedHat makes a lot of things opt-in via RPMs rather than something like symlinks.

Reply:
Yea looks like apache2 ships with a lot of `.so` libraries.

![image](https://user-images.githubusercontent.com/4874123/135530682-da245f3d-6cd7-4e5e-875d-c52806f25be0.png)